### ISSUE TITLE
./configure: error: no /tmp/ngx_pagespeed-1.11.33.4-beta/config was found

### ISSUE THREAD

**Issue by zhex900** [markhilton/docker-nginx-php-fpm#1]
Hi,

The build failed here

./configure: error: no /tmp/ngx_pagespeed-1.11.33.4-beta/config was found

**Comment by markhilton** [comment]
I've decided to maintain separate repositories for Nginx & PHP-FPM containers. Please use those instead: 

- https://github.com/markhilton/docker-php-fpm
- https://github.com/markhilton/docker-nginx-pagespeed

I've just built those with latest Nginx, PageSpeed and PHP-FPM 7.1 versions. You can use public Docker images: 

- https://hub.docker.com/r/crunchgeek/php-fpm/
- https://hub.docker.com/r/crunchgeek/nginx-pagespeed/

Hope that helps, those are not lightweight images, rather all-in-ready-to-go.

**Comment by zhex900** [comment]
Thank you for sharing your work!

Do you mind give me an example of your docker-compose where you use link
both nginx and php-fpm

**Comment by markhilton** [comment]
absolutely, please take a look here: https://github.com/markhilton/larakube
Raw docker-compose.yml example I use with Laravel projects:
https://raw.githubusercontent.com/markhilton/larakube/master/docker-compose.yml
